src/Compilers/Core/Portable/Syntax/SyntaxNodeOrTokenList.cs

```diff
@@ -159,7 +145,7 @@ public TextSpan Span
         /// </returns>
         public override string ToString()
         {
-            return _node != null ? _node.ToString() : string.Empty;
+            return _node?.ToString() ?? string.Empty;
```

Comment:
Theoretically, this does not exactly preserves the previous behavior, but it would be wrong for `ToString()` to return null.